lib/async.js

```diff
@@ -68,6 +68,14 @@
         return memo;
     };
 
+    var _forEachOf = function (object, iterator) {
+        for (key in object) {
```

Comment:
No reason, I kept the original implementation.  I was focused more on merge conflicts.   `_keys` would be faster here...